pkg/rtc/mediatracksubscriptions.go

```diff
@@ -850,26 +718,18 @@ func (t *MediaTrackSubscriptions) stopMaxQualityTimer() {
 	}
 }
 
-func (t *MediaTrackSubscriptions) maybeNotifyNoSubscribers() {
```

Comment:
Removing this function as it is used by remote media track only. And it is a bit awkward. Going to look at remote media track again to avoid use of this and figure out a better way.